bin/www

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+const app = require('../app');
+const port = 3000;
```

Comment:
I think we need a different port here, CRA defaults to 3000

Reply:
Any foreseeable issue if I use 4200?

Reply:
I can't say for sure:) I've used 5000 in the past, I think any port that is not in use will work